src/index.njk

```diff
@@ -26,6 +26,20 @@ masthead: true
         <p class="govuk-body govuk-!-margin-bottom-0"><a href="/patterns/" class="govuk-link govuk-!-font-weight-bold">Browse patterns</a></p>
       </div>
 
+      <div class="govuk-grid-column-full">
+        <hr class="govuk-section-break govuk-section-break--visible govuk-section-break--xl">
+      </div>
+
+      <div class="govuk-grid-column-two-thirds govuk-!-mb-r4">
+        <h2 class="govuk-heading-l">Principles we follow</h2>
+        <p class="govuk-body">
+The GOV.UK Design System helps service teams follow the <a class="govuk-link" href="https://www.gov.uk/guidance/government-design-principles">Government Design Principles</a> and <a class="govuk-link" href="https://www.gov.uk/service-manual">GOV.UK Service Manual</a>. This website also follows the Government Digital Service (GDS) <a class="govuk-link" href="https://www.gov.uk/guidance/style-guide">style guide</a> and <a class="govuk-link" href="https://www.gov.uk/guidance/content-design">content design guidance</a>.
+        </p>
+
+      <p class="govuk-body">
+<a class="govuk-link" href="/community/accessibility-strategy/">Our accessiblity strategy</a> outlines our principles and work to improve accessibility within the GOV.UK Design System.</p>
+      </div>
+
       <div class="govuk-grid-column-full">
         <hr class="govuk-section-break govuk-section-break--visible govuk-section-break--xl govuk-!-margin-top-0">
```

Comment:
```suggestion
        <hr class="govuk-section-break govuk-section-break--visible govuk-section-break--xl">
```

Not sure where it came from, but I'd remove the `govuk-!-margin-top-0` class cos it's squishing the border a bit close to the text above

Reply:
I've moved it to what I think is the proper location. Closing this comment.